commander/src/utils/flags.ts

```diff
@@ -53,6 +53,11 @@ const prettyDescription = 'Prints JSON in pretty format rather than condensed.';
 
 const outputDescription = 'The output directory. Default will set to current working directory.';
 
+const transactionParametersInputFile = `Transaction parameters input file.
```

Comment:
I think this description is quite specific to the transaction creation.
Maybe we can just define it in commander/src/bootstrapping/commands/transaction/create.ts

Reply:
Would making the description more generic work and keep it in the utils where it can be reused?

Reply:
Yes i think that is fine as well. but in that case i think the description should be purely about `file`

Reply:
Yes, absolutely.

Reply:
Done!